server/config/subject_page.proto

```diff
@@ -261,6 +261,40 @@ message MapTileSpec {
   string geo_json_prop = 3;
 }
 
+// Spec for getting value from property values api call
+message PropertySpec {
+  string property = 1;
+  string direction = 2;
+  string type = 3;
+}
+
+// Spec for just displaying a value
+message DisplayValueSpec {
+  repeated string value = 1;
+  repeated string source = 2;
+}
+
+message AnswerMessageTileSpec {
+  oneof answer {
+    PropertySpec property = 1;
+    DisplayValueSpec display_value = 2;
+  }
+  string entity = 3;
+}
+
+message AnswerTableTileSpec {
+  message TableColumn {
+    string header = 1;
+    PropertySpec property = 2;
+  }
+  repeated TableColumn column = 1;
+  repeated string entity = 2;
+}
+
+message EntityOverviewTileSpec {
+  string entity = 1;
```

Comment:
Are we not having entity_ids in the page metadata?  I thought this would be an override/customization.  (But I also don't want to go off into long discussions, so if that was what decided, fine, document well)

Reply:
hmm, then do we want to add this to the Tile message, as a repeated `entity`.  We can then set this for places (they can use the same field) and switch over.

(if we agree, switchover in later PR is OK)

Reply:
done, switched it in this PR